database/seed.js

```diff
@@ -12,10 +12,13 @@ const seed = async () => {
 
   const photos = await listPhotos();
 
-  //loop 1000 times, in each loop, loop through photos url, get applicable data points, and seed db
+  // loop 1000 times, in each loop, loop through photos url, get applicable data points, and seed db
   photos.Contents.forEach(photo => {
-    const url = `https://sdc-airbnb-photos.s3.us-east-2.amazonaws.com/${photo.Key}`
-    console.log(url)
+    const storage_url = `https://sdc-airbnb-photos.s3.us-east-2.amazonaws.com/${photo.Key}`
+    const name = photo.Key.split('.')[0];
```

Comment:
Why is the .split() needed?

Reply:
`photo.Key` returns a name that ends in .jpg.  I need that for url, but just using the file name without extension for the name.